frontend/src/components/funding/TopupFundingCard.vue

```diff
@@ -0,0 +1,115 @@
+                    <v-col cols="12" sm="4">
+                      <strong>${{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeHREmployerHealthTax) }}</strong>
+                    </v-col>
+                  </v-row>
+                  <v-row no-gutters class="my-1">
+                    <v-col cols="12" sm="8">> Professional Development Hours</v-col>
+                    <v-col cols="12" sm="4">
+                      <strong>${{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeHRProDevHours) }}</strong>
+                    </v-col>
+                  </v-row>
+                  <v-row no-gutters class="my-1">
+                    <v-col cols="12" sm="8">> Professional Development Expenses</v-col>
+                    <v-col cols="12" sm="4">
+                      <strong>${{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeHRProDevExpenses) }}</strong>
+                    </v-col>
+                  </v-row>
+                </div>
+              </v-card>
+            </v-col>
+
+            <v-col cols="12" lg="6" class="mb-sm-4 mb-lg-0">
+              <v-row no-gutters>
+                <v-col cols="12" sm="6">
+                  <v-card class="pa-4 mt-2 mt-sm-4 mt-lg-0 mr-sm-2 ml-lg-2">
+                    <h5 class="mb-4">Programming</h5>
+                    <h3>$ {{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeProgramming) }}</h3>
+                  </v-card>
+                </v-col>
+                <v-col cols="12" sm="6">
+                  <v-card class="pa-4 mt-4 mt-lg-0 ml-sm-2 ml-lg-4">
+                    <h5 class="mb-4">Administrative</h5>
+                    <h3>$ {{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeAdministrative) }}</h3>
+                  </v-card>
+                </v-col>
+                <v-col cols="12" sm="6">
+                  <v-card class="pa-4 mt-4 mt-lg-6 ml-lg-2 mr-sm-2">
+                    <h5 class="mb-4">Facility</h5>
+                    <h3>$ {{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeFacility) }}</h3>
+                  </v-card>
+                </v-col>
+                <v-col cols="12" sm="6">
+                  <v-card class="pa-4 mt-4 mt-lg-6 ml-sm-2 ml-lg-4">
+                    <h5 class="mb-4">Operational</h5>
+                    <h3>$ {{ format.formatDecimalNumber(fundingDetails?.topupEnvelopeOperational) }}</h3>
+                  </v-card>
+                </v-col>
+              </v-row>
+            </v-col>
+          </v-row>
+        </v-container>
+      </v-skeleton-loader>
+    </v-card>
+  </div>
+</template>
+
+<script>
+import format from '@/utils/format'
+
+export default {
+  name: 'TopupFundingCard',
+  props: {
+    loading: {
+      type: Boolean,
+      default: true,
+    },
+    fundingDetails: {
+      type: Object,
+      required: true,
+      default: () => {
+        return {}
+      },
+    },
+  },
+  computed: {
+    shouldShowCard() {
```

Comment:
I think the logic to determine whether we show the TopUpFunding card or not should be determined in the parent. 

The logic how it is now, will always render the TopUpCard component, even if we don't need it. 

so we should move the v-if into line 13 of FundingAllocationTab.